Task9.1.js

```diff
@@ -1,6 +1,8 @@
 'use strict'
 
+
 function lastVariant () {
+    let input = document.querySelector('.btn');
```

Comment:
@Lu-el 
Well, not too bad for the first attempt =)
But there are some things that needs to be improved:
1. It wont work with `?!:;,.` symbols
2. I'd suggest you (before removing repeating characters from words) create 2 arrays: the first one for space, tabs and `?!:;,.` symbols, and the second array for words
3. repeating characters is better to detect using `indexOf` and `lastIndexOf` string methods